### DISCUSSION TITLE
SL and TP based on high/low of candle

### DISCUSSION THREAD

**Discussion by rinkesh-ahuja** [kernc/backtesting.py#615] [Q&A]
Can we set the SL at the low of the 'signal' candle? And then set the TP based on the risk taken (entry price - SL) * 2 or 3 or whatever?

Thank you very much!

**Comment by kernc** [comment]
E.g.
```py
        c, l = self.data.Close[-1], self.data.Low[-1]
        self.buy(limit=c, sl=l, tp=c + 3*(c - l))
```

**Reply by samueltg92** [reply]
Hey @kernc is there a way using the code you provided above to plot how the SL and TP is being set?

**Reply by kernc** [reply]
Why not use the search! See: https://github.com/kernc/backtesting.py/issues/56.
